tests/recommend/recommned_test.go

```diff
@@ -0,0 +1,81 @@
+package recommend_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/accuknox/auto-policy-discovery/src/recommendpolicy"
+	"github.com/kubearmor/KubeArmor/tests/util"
+)
+
+func checkPod(name string, ant string, ns string) {
+	pods, err := util.K8sGetPods(name, ns, []string{ant}, 60)
+	Expect(err).To(BeNil())
+	Expect(len(pods)).To(Equal(1))
+}
+
+var _ = BeforeSuite(func() {
+
+	// create new namespace
+	_, err := util.Kubectl(fmt.Sprintf("create namespace rs-demo"))
+	Expect(err).To(BeNil())
+
+	// install replicaset
+	err = util.K8sApply([]string{"conf/replicaset.yaml"})
+	Expect(err).To(BeNil())
+
+	// check for replicaset pod
+	checkPod("nginx-rs-",
+		"container.apparmor.security.beta.kubernetes.io/nginx: localhost/kubearmor-rs-demo-nginx-rs-nginx", "rs-demo")
+
+	// install statefulset
+	err = util.K8sApply([]string{"conf/statefulset.yaml"})
+	Expect(err).To(BeNil())
+
+	// check for statefulset pod
+	checkPod("my-statefulset-",
+		"container.apparmor.security.beta.kubernetes.io/my-container: localhost/kubearmor-default-my-statefulset-my-container", "default")
+})
+
+var _ = AfterSuite(func() {
+
+	// create new namespace
+	_, err := util.Kubectl(fmt.Sprintf("delete namespace rs-demo"))
+	Expect(err).To(BeNil())
```

Comment:
can we have both the sts and rs on the same namespace and make sure that they are deleted after the test is finished?
Currently the sts is not getting deleted after test finishes. @Prateeknandle